documentation/api/query/v4/event-counts.markdown

```diff
@@ -60,7 +61,13 @@ This endpoint builds on top of the [`events`][events] endpoint, and supports all
 ### Response format
 
 The response is a JSON array of maps. Each map contains the counts of events that matched the input
-parameters. The events are counted based on their statuses: `failures`, `successes`, `noops`, `skips`.
+parameters. The events are counted based on their statuses and corrective_change flags (PE only):
+`failures` (status = 'failure'),
+`successes` (status = 'success' and corrective_change = false),
+`noops` (status = 'noop' and corrective_change = false),
```

Comment:
FOSS users will never have corrective_change = false, so we might want to make this a little bit more clear. I also think it would be preferable to return null for the corrective_* fields in FOSS rather than 0, since 0 isn't really the truth (they may well have corrective noops, but they won't be reflected in the count). I understand that you can't get nulls out of postgres's count function -- might be worth some kind of special flag for this.